source/modes/MenuModeConfigureSeats.cpp

```diff
@@ -160,12 +160,7 @@ void MenuModeConfigureSeats::activate()
                 CEGUI::ListboxTextItem* item = new CEGUI::ListboxTextItem(faction, cptFaction);
```

Comment:
They need feed-back, so I'll try to contact them about this in a few days once I've got used to the new feature.
Did you guys had fun with the LAN? :)